packages/server/src/integrations/firebase.ts

```diff
@@ -30,12 +31,21 @@ const SCHEMA: Integration = {
     },
     privateKey: {
       type: DatasourceFieldType.STRING,
+      display: "Private Key",
       required: true,
     },
     projectId: {
       type: DatasourceFieldType.STRING,
+      display: "Project ID",
       required: true,
     },
+    databaseId: {
+      type: DatasourceFieldType.STRING,
+      display: "Database ID",
+      required: false,
+      default: "(default)",
```

Comment:
NAB: Should this be `"default"` instead of `"(default)"`

Reply:
it seems to be specifically `(default)` in firestore

`default` won't connect but `(default)` and my second db named `second-db` both work

![image](https://github.com/user-attachments/assets/19768d57-b7ea-449b-9cf6-44e625e7a3b8)